docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md

```diff
@@ -0,0 +1,142 @@
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+When you've completed this guide, you should be able to load and log into LORIS in your browser. Further setup to customize LORIS for your project will be required after that. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to complete the setup for LORIS.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.3) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, the following package manager are required to build LORIS:
+* NodeJS 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* curl
+* zip
+* unzip
+* php-json
+* make
+* software-properties-common
+* php7.3-mysql
+* php7.3-xml
+* php7.3-mbstring
+* php7.3-gd
+* php7.3-zip
+* php7.3-curl (for development instances only)
+* libapache2-mod-php7.3
+
+## Getting the source code
+
+Visit the [releases](github.com/aces/loris/releases) page and download the zipped file of the latest LORIS
+release. The below command will download the latest LORIS release and save 
+it to a file named `loris-src.tar.gz`.
+
+```bash
+curl -s https://api.github.com/repos/aces/loris/releases/latest \
+| grep 'tarball' \
+| cut -d : -f2,3 \
+| tr -d \" \
+| tr -d ,\
+| xargs -n 1 curl -s -o loris-src.tar.gz
+```
+
+When this is complete, expand the compressed file and move its contents to the
+directory from which LORIS will be served. 
+
+```bash
+tar -zxf loris-src.tar.gz
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+```bash
+mv ./loris /var/www/; cd /var/www/loris
+```
+
+Once in the correct directory, run one of the following commands given your environment:
+
+```bash
+make      // For production environments
```

Comment:
```suggestion
make     # For production environments
```
Looking at the new section following, and given this is bash -- let's use `#` instead of `//` for comments in this doc.